src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,8 +1,26 @@
-export const TodoFilter = () => (
+import { Visible } from '../../types/Visible';
+
+type Props = {
+  visible: Visible;
+  setVisible: (arg: Visible) => void;
+  filterBy: string;
+  setFilterBy: (arg: string) => void;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  visible = 2,
```

Comment:
The `visible` prop is initialized with a default value of `2`, which is a number. However, `visible` should be of type `Visible`, which likely means it should be a string like 'all', 'active', or 'completed'. Please update this default value to match the expected type.